servers.json

```diff
@@ -1,4 +1,12 @@
 [
+     {
+        "id": "visionmc",
+        "name": "VisionMC Network",
+        "addresses": [
+            "visionmc.club"
+            "na.visionmc.club"
```

Comment:
Please look over the checklist again.

Reply:
This is what we wanted to be added the others were a mix up this one is correct.

Reply:
It seems fine I do not see any issue?

Reply:
All good